.ci/jobs/defaults.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+##### GLOBAL METADATA
+
+- meta:
+    cluster: apm-ci
+
+##### JOB DEFAULTS
+
+- job:
```

Comment:
@jonahbull I miss a reference to the location of the Jenkinsfile

`script-path: 'Jenkinsfile'`

Reply:
Good catch, @kuisathaverat I'll push a fix shortly.

Reply:
Added that item and a few others that got missed when flattening the jobs originally and corrected the flattening script to warn us when there are items that were present originally missing in the flattened job.